specification/cognitiveservices/OpenAI.Inference/models/audio/commons.tsp

```diff
@@ -0,0 +1,140 @@
+For more output data use 'verbose_json.
+""")
+@added(ServiceApiVersions.v2023_09_01_Preview)
+model AudioTranscriptionOptionsCommon {
+    @doc("""
+    The content type for the operation. Always multipart/form-data for this operation.
+    You need to set your content separtor with the boundary value hardcode here: "azure_openai"
+    """)
+    @header("content-type")
+    contentType: "multipart/form-data; boundary=azure_openai";
+
+    @doc("The content length of the operation. This needs to be provided by the caller.")
+    @header("content-length")
+    contentLength: int64;
+
+    @doc("The audio file object to transcribe.")
+    file: bytes;
+
+    @doc("An optional text to guide the model's style or continue a previous audio segment. The prompt should match the audio language.")
+    prompt?: string;
+
+    @doc("""
+    The sampling temperature, between 0 and 1.
+    Higher values like 0.8 will make the output more random, while lower values like 0.2 will make it more focused and deterministic.
+    If set to 0, the model will use log probability to automatically increase the temperature until certain thresholds are hit.
+    """)
+    temperature?: float32 = 0;
+
+    @doc("""
+    The model name to provide as part of this transcription request.
+    Not applicable to Azure OpenAI, where deployment information should be included in the Azure
+    resource URI that's connected to.
+    """)
+    @projectedName("json", "model")
+    @projectedName("csharp", "InternalNonAzureModelName")
+    `model`?: string;
+}
+
+@doc("Transcription response containing only the transcribed text.")
+@added(ServiceApiVersions.v2023_09_01_Preview)
+model AudioTranscriptionSimpleJson {
+    @doc("Transcribed text.")
+    text: string;
+}
+
+@doc("Audio transcription task type")
+@added(ServiceApiVersions.v2023_09_01_Preview)
+enum AudioTranscriptionTask {
+    @doc("Value for a transcription task")
+    transcribe: "transcribe",
+
+    @doc("Value for a translation task")
+    translate: "translate",
+}
+@doc("Transcription response.")
+@added(ServiceApiVersions.v2023_09_01_Preview)
+model AudioTranscriptionVerboseJson extends AudioTranscriptionSimpleJson {
+    @doc("Audio transcription task.")
+    task: AudioTranscriptionTask;
+
+    @doc("Language detected in the source audio file.")
+    language: string;
+
+    @doc("Duration.")
+    @encode("seconds", float32)
+    duration: duration;
+
+    @doc("Segments.")
+    segments: AudioTranscriptionSegment[];
+}
+
+@doc("Transcription segment.")
+@added(ServiceApiVersions.v2023_09_01_Preview)
+model AudioTranscriptionSegment {
+    @doc("Segment identifier.")
+    id: int32;
+
+    @doc("Segment start offset.")
+    start: float32;
+
+    @doc("Segment end offset.")
+    end: float32;
+
+    @doc("Segment text.")
+    text: string;
+
+    @doc("Temperature.")
+    temperature: float32;
+
+    @doc("Average log probability.")
+    @projectedName("json", "avg_logprob")
+    averageLogProb: float32;
```

Comment:
shouldn't this be `avgLogprob`? If we need a different name for a specific language, we can add it as a projectedName for that language?